src/components/routes/homepage/MediaCard.js

```diff
@@ -1,77 +1,93 @@
   CardContent,
+  Container,
   Typography,
-  makeStyles,
-  ExpansionPanel,
-  ExpansionPanelSummary,
-  ExpansionPanelDetails
+  makeStyles
 } from '@material-ui/core';
+import { FavoriteBorder } from '@material-ui/icons';
+import placeholder from '../../../assets/no_image_placeholder.png';
 
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
-
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   root: {
-    maxWidth: 345,
-    padding: '10px',
-    fontFamily: "'Merriweather', serif"
+    width: 'min(450px, 90%)',
+    fontFamily: "'Merriweather', serif",
+    margin: 10,
+    '&:hover': {
+      cursor: 'pointer'
+    },
+    textAlign: 'left'
+  },
+  buttonsBar: {
+    textAlign: 'right',
+    padding: 10
+  },
+  button: {
+    margin: theme.spacing(1),
+    width: 60,
+    '&:hover': {
+      backgroundColor: 'inherit'
+    }
+  },
+  content: {
+    margin: 0,
+    padding: `0px ${theme.spacing(2)}px`
+  },
+  date: {
+    color: theme.palette.primary.main,
+    fontWeight: 500
+  },
+  title: {
+    color: theme.palette.primary.dark,
+    fontWeight: 800
+  },
+  venue: {
+    color: '#9e9e9e'
   },
   media: {
-    fontFamily: "'Merriweather', serif",
     height: 0,
     paddingTop: '56%'
-  },
-  expansionPanel: {
-    lineHeight: '0px',
-    marginBottom: '0px',
-    fontFamily: "'Merriweather', serif"
   }
-});
+}));
 
 export const MediaCard = ({ event }) => {
+  const [elevation, setElevation] = useState(1);
   const classes = useStyles();
-  const {
-    title,
-    start,
-    end,
-    venue,
-    description,
-    image_url,
-    organiser: { username, email }
-  } = event;
+
+  const { title, start, venue, image_url } = event;
 
   return (
-    <Card className={classes.root}>
-      <CardActionArea>
-        <CardMedia className={classes.media} image={image_url} title={title} />
-        <CardContent className={classes.root}>
-          <h4>{title}</h4>
-          <Typography variant={'body2'}>
-            Date:{' '}
-            {`${moment(start).format('lll')} - ${moment(end).format('lll')}`}{' '}
-            <br />
-            Venue: {venue} <br />
-            Contact: {`${username} (${email})`}
-          </Typography>
-        </CardContent>
-      </CardActionArea>
-      <ExpansionPanel className={classes.expansionPanel}>
-        <ExpansionPanelSummary
-          expandIcon={<ExpandMoreIcon />}
-          aria-controls="panel1a-content"
-          id="panel1a-header"
-        >
-          Learn More
-        </ExpansionPanelSummary>
-        <ExpansionPanelDetails>
-          <Typography>{description}</Typography>
-        </ExpansionPanelDetails>
-      </ExpansionPanel>
+    <Card
+      className={classes.root}
+      elevation={elevation}
+      onMouseOver={() => setElevation(5)}
+      onMouseOut={() => setElevation(1)}
+    >
+      <CardMedia
+        className={classes.media}
+        image={image_url ? image_url : placeholder}
+        title={title}
+      />
```

Comment:
has this been tested with a sample image from the internet? My implementation for the user profile picture is basically the same but I had to do this to render an image url:
```
<CardMedia
  src={image_url}
  component='img'
/>
```
Then I think the placeholder has to be rendered like this:
`<CardMedia image={placeholder} />`

placeholder image is a proper `.png | .jpg | ...` file whereas `image_url` is a string pointing to a http / https url

Reply:
it has been tested, works well with images from `https://rcltech-raven.s3.aws...`

so my purpose of doing this is : 

when upload to S3 fails, `phoenix` will still create an event and put an empty string in the `image_url` field. the ternary operator is there in case this happen and phoenix return an empty string (empty string will give you a `false` value I think).

Reply:
I see. I will double check my implementation.